tofu/_plot.py

```diff
@@ -87,8 +87,8 @@ def _plot_shotoverview_init(ns=1, sharet=True, sharey=True, shareRZ=True,
         else:
             laxt[ii] = fig.add_subplot(axarr[ii,:2], sharex=sht, sharey=shy)
             laxc[ii] = fig.add_subplot(axarr[ii,2], sharex=shRZ, sharey=shRZ)
-            if not shareRZ:
-                ax2.set_aspect('equal', adjustable='datalim')
+            # if not shareRZ:
```

Comment:
Reason for this commenting ? was it the source of a bug ?

Reply:
Yes, sorry I will uncomment it as I treated in the PR: #456

Reply:
(`ax2` is undefined)